dataikuapi/dss/ml.py

```diff
@@ -985,26 +1012,37 @@ def __repr__(self):
             return "DSSSubpopulationCategoryModalityDefinition(%s='%s')" % (self.feature_name, self.value)
 
 
-class DSSSubpopulationAnalysis(DSSInternalDict):
+class DSSSubpopulationAnalysis(object):
     """
     Object to read details of a subpopulation analysis of a trained model
 
     Do not create this object directly, use :meth:`DSSSubpopulationAnalyses.get_analysis(feature)` instead
     """
 
     def __init__(self, analysis, prediction_type):
-        super(DSSSubpopulationAnalysis, self).__init__(analysis)
-        self.computed_as_type = self.get("computed_as_type")
-        self.modalities = [DSSSubpopulationModality(analysis.get("feature"), self.computed_as_type, m, prediction_type) for m in self.get("modalities", [])]
+        self._internal_dict = analysis
+        self.computed_as_type = analysis["computed_as_type"]
+        self.modalities = [DSSSubpopulationModality(analysis["feature"], self.computed_as_type, m, prediction_type) for m in analysis.get("modalities", [])]
+
+    def get_raw(self):
+        """
+        Gets the raw dictionary of the actual data
+
+        :rtype: dict
+        """
+        return self._internal_dict
+
+    def __repr__(self):
+        return self.__class__.__name__ + "(" + self._internal_dict.__repr__() + ")"
```

Comment:
Ditto, can maybe indicate the column name, the type and the number of modalities